pages/dkp/konvoy/2.2/image-builder/create-azure-image/index.md

```diff
@@ -0,0 +1,70 @@
+
+## Prerequisites
+
+Before you begin, you must:
+
+-   Download the latest [Konvoy Image Builder](https://github.com/mesosphere/konvoy-image-builder/releases) bundle (prefixed with `konvoy-image-bundle`) for your OS. Do not use the release prefixed with `konvoy-image-builder`.
+
+-   Create a working `Docker` setup.
+
+## Extract Bundle
+
+Extract the bundle and `cd` into the extracted `konvoy-image-bundle-$VERSION_$OS` folder. The bundled version of `konvoy-image` contains an embedded `docker` image that contains all the requirements for building.
+
+<p class="message--note"><strong>NOTE: </strong> The <code>konvoy-image</code> binary and all supporting folders are also extracted. When run, <code>konvoy-image</code> bind mounts the current working directory (<code>${PWD}</code>) into the container to be used.</p>
+
+-   Set environment `AZURE_CLIENT_SECRET` variable:
+
+    ```bash
+    export AZURE_CLIENT_SECRET
+    ```
+
+-   Ensure you have an [override file](../override-files) to configure specific attributes of your Azure image.
+
+## Build the image
+
+Run the `konvoy-image` command to build and validate the image.
+
+```sh
+konvoy-image build --client-id <azure_client_id> --tenant-id <azure_tenant_id> --overrides override-source-image.yaml --overrides override-images.yaml images/azure/centos-7.yaml
+```
+
+By default it builds in the `westus2` location. To specify another location set the `--location` flag:
+
+```sh
+konvoy-image build --client-id <azure_client_id> --tenant-id <azure_tenant_id> --location eastus --overrides override-source-image.yaml --overrides override-images.yaml images/ami/centos-7.yaml
+```
+
+When the command is complete the image id is printed and written to `./manifest.json`.
+
+## Image Gallery
+
+By default Konvoy Image Builder will create a Resource Group, Gallery, and Image Name to store the resulting image in. To specify a specific Resource Group, Gallery, or Image Name flags may be specified:
+
+```sh
+--gallery-image-locations location   a list of locatins to publish the image (default same as location)
+--gallery-image-name string          the gallery image name to publish the image to
+--gallery-image-offer string         the gallery image offer to set (default "dkp")
+--gallery-image-publisher string     the gallery image publisher to set (default "dkp")
+--gallery-image-sku string           the gallery image sku to set
+--gallery-name string                the gallery name to publish the image in (default "dkp")
+--resource-group string              the resource group to create the image in (default "dkp")
+```
+
+The SKU and Image Name will default to the values found in the image yaml.
+
+<!--- ## Air Gapped
```

Comment:
When we get there, this air gapped section needs to be its own topic, not in this file.

Reply:
totes, removed.